tools/code_format/check_format_go_code.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -e
+
+tools="$(dirname "$(dirname "$(realpath "$0")")")"
+root=$(realpath "$tools/..")
+
+cd "$root" || exit 1
+
+gofmt="@go_sdk//:bin/gofmt"
```

Comment:
```suggestion
GOFMT_BIN="bazel run @go_sdk//:bin/gofmt -- "
```
if the intention is to make the gofmt configurable in some way then probably use a bash global explicitly and the full bazel command so that you can override it without bazel